src/FormikContext.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { FormikContext } from './types';
+
+const PrivateFormikContext = React.createContext<FormikContext<any>>({} as any);
```

Comment:
This question might be a bit of a naive question but how will formik support multiple forms? Won't the context always be the same and therefore rendering two forms will cause data to bleed?

Reply:
No, context only applies to subtree which is wrapped with the provider.

Reply:
Thanks